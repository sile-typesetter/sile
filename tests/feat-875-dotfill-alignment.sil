\begin[papersize=a5]{document}
\script[src=packages/leaders]
\nofolios
\neverindent
% Test case: vertical alignment for \dotfill
\font[size=50pt]
|\dotfill{}|\par
1\dotfill{}1\par
1a\dotfill{}1a\par
1 2\dotfill{}2 1\par
1 2a\dotfill{}2a 1\par
1 2 3 4\dotfill{}4 3 2 1\par
1 2 3 4 5\dotfill{}5 4 3 2 1\par
1 2 3 4 5 |\dotfill{}5 4 3 2 1\par
\end{document}
