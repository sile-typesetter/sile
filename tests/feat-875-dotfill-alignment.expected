Set paper size 	419.5275636	595.275597
Begin page
Mx 	20.9764
My 	70.5353
Set font 	Gentium Plus;50;400;;normal;;LTR
T	95 w=12.2070	(|)
Mx 	34.0492
Mx 	46.5492
T	17 w=11.4502	(.)
Mx 	70.4994
Mx 	82.9994
T	17 w=11.4502	(.)
Mx 	106.9496
Mx 	119.4496
T	17 w=11.4502	(.)
Mx 	143.3998
Mx 	155.8998
T	17 w=11.4502	(.)
Mx 	179.8500
Mx 	192.3500
T	17 w=11.4502	(.)
Mx 	216.3002
Mx 	228.8002
T	17 w=11.4502	(.)
Mx 	252.7504
Mx 	265.2504
T	17 w=11.4502	(.)
Mx 	289.2006
Mx 	301.7006
T	17 w=11.4502	(.)
Mx 	325.6508
Mx 	338.1508
T	17 w=11.4502	(.)
Mx 	386.3442
T	95 w=12.2070	(|)
Mx 	20.9764
My 	130.5353
T	20 w=23.4619	(1)
Mx 	70.4994
Mx 	82.9994
T	17 w=11.4502	(.)
Mx 	106.9496
Mx 	119.4496
T	17 w=11.4502	(.)
Mx 	143.3998
Mx 	155.8998
T	17 w=11.4502	(.)
Mx 	179.8500
Mx 	192.3500
T	17 w=11.4502	(.)
Mx 	216.3002
Mx 	228.8002
T	17 w=11.4502	(.)
Mx 	252.7504
Mx 	265.2504
T	17 w=11.4502	(.)
Mx 	289.2006
Mx 	301.7006
T	17 w=11.4502	(.)
Mx 	325.6508
Mx 	338.1508
T	17 w=11.4502	(.)
Mx 	375.0893
T	20 w=23.4619	(1)
Mx 	20.9764
My 	190.5353
T	20 68 w=46.4111	(1a)
Mx 	70.4994
Mx 	82.9994
T	17 w=11.4502	(.)
Mx 	106.9496
Mx 	119.4496
T	17 w=11.4502	(.)
Mx 	143.3998
Mx 	155.8998
T	17 w=11.4502	(.)
Mx 	179.8500
Mx 	192.3500
T	17 w=11.4502	(.)
Mx 	216.3002
Mx 	228.8002
T	17 w=11.4502	(.)
Mx 	252.7504
Mx 	265.2504
T	17 w=11.4502	(.)
Mx 	289.2006
Mx 	301.7006
T	17 w=11.4502	(.)
Mx 	352.1401
T	20 68 w=46.4111	(1a)
Mx 	20.9764
My 	250.5353
T	20 w=23.4619	(1)
Mx 	57.6512
T	21 w=23.4619	(2)
Mx 	106.9496
Mx 	119.4496
T	17 w=11.4502	(.)
Mx 	143.3998
Mx 	155.8998
T	17 w=11.4502	(.)
Mx 	179.8500
Mx 	192.3500
T	17 w=11.4502	(.)
Mx 	216.3002
Mx 	228.8002
T	17 w=11.4502	(.)
Mx 	252.7504
Mx 	265.2504
T	17 w=11.4502	(.)
Mx 	289.2006
Mx 	301.7006
T	17 w=11.4502	(.)
Mx 	338.4145
T	21 w=23.4619	(2)
Mx 	375.0893
T	20 w=23.4619	(1)
Mx 	20.9764
My 	310.5353
T	20 w=23.4619	(1)
Mx 	57.6512
T	21 68 w=46.4111	(2a)
Mx 	106.9496
Mx 	119.4496
T	17 w=11.4502	(.)
Mx 	143.3998
Mx 	155.8998
T	17 w=11.4502	(.)
Mx 	179.8500
Mx 	192.3500
T	17 w=11.4502	(.)
Mx 	216.3002
Mx 	228.8002
T	17 w=11.4502	(.)
Mx 	252.7504
Mx 	265.2504
T	17 w=11.4502	(.)
Mx 	315.4652
T	21 68 w=46.4111	(2a)
Mx 	375.0893
T	20 w=23.4619	(1)
Mx 	20.9764
My 	370.5353
T	20 w=23.4619	(1)
Mx 	57.6512
T	21 w=23.4619	(2)
Mx 	94.3260
T	22 w=23.4619	(3)
Mx 	131.0008
T	23 w=23.4619	(4)
Mx 	179.8500
Mx 	192.3500
T	17 w=11.4502	(.)
Mx 	216.3002
Mx 	228.8002
T	17 w=11.4502	(.)
Mx 	265.0649
T	23 w=23.4619	(4)
Mx 	301.7397
T	22 w=23.4619	(3)
Mx 	338.4145
T	21 w=23.4619	(2)
Mx 	375.0893
T	20 w=23.4619	(1)
Mx 	20.9764
My 	430.5353
T	20 w=23.4619	(1)
Mx 	57.6512
T	21 w=23.4619	(2)
Mx 	94.3260
T	22 w=23.4619	(3)
Mx 	131.0008
T	23 w=23.4619	(4)
Mx 	167.6756
T	24 w=23.4619	(5)
Mx 	228.3901
T	24 w=23.4619	(5)
Mx 	265.0649
T	23 w=23.4619	(4)
Mx 	301.7397
T	22 w=23.4619	(3)
Mx 	338.4145
T	21 w=23.4619	(2)
Mx 	375.0893
T	20 w=23.4619	(1)
Mx 	20.9764
My 	490.5353
T	20 w=23.4619	(1)
Mx 	57.6512
T	21 w=23.4619	(2)
Mx 	94.3260
T	22 w=23.4619	(3)
Mx 	131.0008
T	23 w=23.4619	(4)
Mx 	167.6756
T	24 w=23.4619	(5)
Mx 	204.3504
T	95 w=12.2070	(|)
Mx 	228.3901
T	24 w=23.4619	(5)
Mx 	265.0649
T	23 w=23.4619	(4)
Mx 	301.7397
T	22 w=23.4619	(3)
Mx 	338.4145
T	21 w=23.4619	(2)
Mx 	375.0893
T	20 w=23.4619	(1)
End page
Finish
